router/core/src/main/scala/com/twitter/finagle/naming/buoyant/RichNoBrokersAvailableException.scala

```diff
@@ -34,43 +31,6 @@ ${formatDtab(path.localDtab)}
 
 object RichNoBrokersAvailableException {
 
-  def apply(path: Dst.Path, namer: NameInterpreter): Future[Nothing] = {
```

Comment:
Minor import cleanup since we no longer use the Delegator
```diff
diff --git a/router/core/src/main/scala/com/twitter/finagle/naming/buoyant/RichNoBrokersAvailableException.scala b/router/core/src/main/scala/com/twitter/finagle/naming/buoyant/RichNoBrokersAvailableException.scala
index 26dcd1654..cdcfa4c8d 100644
--- a/router/core/src/main/scala/com/twitter/finagle/naming/buoyant/RichNoBrokersAvailableException.scala
+++ b/router/core/src/main/scala/com/twitter/finagle/naming/buoyant/RichNoBrokersAvailableException.scala
@@ -2,9 +2,7 @@ package com.twitter.finagle.naming.buoyant
 
 import com.twitter.finagle._
 import com.twitter.finagle.buoyant.Dst
-import com.twitter.finagle.naming.NameInterpreter
 import com.twitter.util.Future
-import io.buoyant.namer.{DelegateTree, Delegator, RichActivity}
 
 class RichNoBrokersAvailableException(
   path: Dst.Path,

```